packages/blocks/src/resolveBlockTemplate.ts

```diff
@@ -0,0 +1,40 @@
+import find from 'lodash/find.js';
```

Comment:
Nice! Does this only add lodash's `find` function to the public bundle?

Reply:
Yes this is the preffered way to import lodash.